src/entities/catalogs/services/get-videos-by-catalog.ts

```
@@ -333,25 +318,6 @@ async function getPageviewByCatalogId(catalogId: string): Promise<number> {
-
-  // Map through the rows and create an object with id and value of the catalog
-  return response.rows.map((row) => {
-    if (row.dimensionValues && row.metricValues) {
-      return {
-        id: row?.dimensionValues[0]?.value,
-        pageviews: parseInt(row?.metricValues[0]?.value ?? "0", 10),
-      };
-    }
-  });
+function createPlaylistId(channelId: string) {
+  return channelId.substring(0, 1) + "U" + channelId.substring(2);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)